### PR TITLE
Expand peer dependency range for nextjs 12

### PR DESCRIPTION
**Pull request by elmarburke**
The package works well with next 12, it's only not listed in the peerDependencies. This PR widens the peerDependencies to support also v12

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
   },
   "peerDependencies": {
     "@sanity/client": "^2.11.0",
-    "next": "^11.0.0",
+    "next": "^11.0.0 ^12.0.0",
```
Comment (lorenzodejong) [inline package.json:46]: Providing multiple package versions is not supported, could you change this to be just `^12.0.0`?

**Review by lorenzodejong — CHANGES_REQUESTED** [review]

**Comment by lorenzodejong** [comment]
Thanks for the PR, however the syntax used in the package.json seems to be incorrect. Could you also install the package to update the package-lock.json and include this in the commit? Thanks!

File: package.json (hunk 1 above)
Comment (elmarburke) [reply package.json]: Sorry, my fault, I forgot to add the logical or `||` for the semver range 🤦 
Only using `^12.0.0` would make it a breaking change for this package and is not particular needed here.

PR is updated to support both.

**Review by lorenzodejong — APPROVED** [review]

**Comment by lorenzodejong** [comment]
It does seem like there are some issues with the upgrade to Next.js 12 (#23). We'll have to wait for this to be resolved before integrating this PR.

**Comment by elmarburke** [comment]
Would you mind publishing a prerelease ([e.g. with a alpha tag](https://survivejs.com/maintenance/packaging/publishing/#publishing-a-pre-release-version)) since it seams like no functionality will break and "only" warnings are emitted to the console? That would help me a lot actually.

**Pull request merged**